website/docs/script/type-id.mdx

```diff
@@ -0,0 +1,237 @@
+---
+id: type-id
+title: "Tutorial: Type ID (Upgradable Script)"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import ScriptHeaders from "./ScriptHeaders.tsx";
+import TutorialHeader from "@components/TutorialHeader";
+import ScriptTools from "./_ScriptTools.mdx";
+
+# Tutorial: Type ID (Upgradable Script)
+
+<TutorialHeader
+  time={ScriptHeaders.typeid.time}
+  topics={ScriptHeaders.typeid.topics}
+  tools={ScriptHeaders.typeid.tools}
+/>
+
+Smart contracts are codes that got executed on-chain that nobody can stops. However, sometimes you might want to implement some kind of upgradability for your smart contract in case there are bugs in the source code.
```

Comment:
```suggestion
Scripts are codes that execute on-chain and cannot be stopped. However, sometimes you might want to ensure your Script is upgradable in case there are bugs in the source code.
```